### DISCUSSION TITLE
Reference documentation subproject formalisation

### DISCUSSION THREAD

**Discussion by divya-mohan0209** [kubernetes/website#43767] [General]
Following the discussion on [this slack thread](https://kubernetes.slack.com/archives/CA1MMM1HU/p1696327021747639?thread_ts=1696327021.747639&cid=CA1MMM1HU), we have identified the need to

- formalise the reference documentation subproject
- staff it appropriately with dedicated OWNERS
- socialise this as an opportunity for more people to get involved with SIG Docs.

Toward this, here's a set of immediate goals that, I think, we should get started with

- [ ] Identification of potential prerequisites & challenges for people while beginning to contribute to this subproject, such as knowledge of Go, understanding the ref doc generation process etc. Please add below, if there's something I've missed.
- [x] Shoutout as an issue that **we** need help with during maintainer track session at KubeCon NA (action owners: @natalisucks and @divya-mohan0209) 
- [ ] Discussion around timelines for formalisation (can be done here)
- [ ] Nominations for staffing owners. Consensus can be sought via our mailing lists & Slack channel. We can socialize them during our meetings.
- [ ] Work with subproject owners to smoothen the learning curve for contributors to this are, if possible.
- [ ] Issue a call-out via Contributor Comms to seek out more contributors for the project.

cc: @reylejano @sftim @onlydole @tengqm @kbhawkey @a-mccarthy @seokho-son

**Comment by sftim** [comment]
I propose that we aim to get the subproject formalised by the end of Q3 2024.
